tap_google_sheets/schema.py

```diff
@@ -19,21 +21,29 @@ def colnum_string(num):
     return string
 
 
+def pad_default_effective_values(headers, first_values):
+    for i in range(len(headers) - len(first_values)):
+        first_values.append(OrderedDict())
+
 # Create sheet_metadata_json with columns from sheet
+
+
 def get_sheet_schema_columns(sheet):
     sheet_title = sheet.get('properties', {}).get('title')
     sheet_json_schema = OrderedDict()
     data = next(iter(sheet.get('data', [])), {})
     row_data = data.get('rowData', [])
-    if row_data == []:
+    if row_data == [] or len(row_data) == 1:
```

Comment:
idk the context of why len == 1 is here but the conditional seems like it might be redundant. Wouldnt <= 1 be the same?

Reply:
Another change pulled in from Master